cylc/flow/tui/app.py

```diff
@@ -310,11 +310,10 @@ def get_snapshot(self):
                 }
             )
         except WorkflowStopped:
-            self.client = None
```

Comment:
I think this is needed. It is what allows Tui to attempt to reconnect at a later time.

Try:

* install
* tui <flow>
* play <flow>
* stop <flow>
* play <flow>
* stop <flow>

Tui should connect/disconnect automatically.

Similarly Tui should connect automatically when the workflow is played in this example:

* tui <flow>
* install
* clean <flow>
* install
* play <flow>

Reply:
Removed by accident along with debugging stuff.

```suggestion
             self.client = None 
```

Reply:
This suggestion was not applied - https://github.com/cylc/cylc-flow/issues/4818